Website Reformat

Masterpoda opened the pull request:
I reformatted the prominent section headers into tabbed pages. We can pretty easily change what the tabs are, and what is shown on each of the tabbed pages in the future. The website is still purely html and css, and it follows the toastmasters style guide, so it should still be pretty robust and not require modern browsers.

Masterpoda committed:
movedsite data into different tab pages

Masterpoda committed:
added better banners and footers, began adding more content.

Masterpoda committed:
added pictures, polished up some other pages.

Masterpoda committed:
added borders to video and google map

Masterpoda committed:
minor formatting change to video.

bugi reviewed (changes requested):
Thank you for dragging the look of the website out of the '90s.

Please do some housekeeping on what you commit.  In particular, remove file "websites/tm375.org/documents/.~lock.Meeting Agenda Template.docx#" and directory ".vscode", and add entries to a .gitignore file so they don't get added again.  While you're in there, you should guard against .DS_Store and other filenames that shouldn't be added to a repo.

The website may be (and is) served from a filesystem that respects file case.  Make sure any references to them reflect that.  I'm not sure whether there are any such problems in the merge, but the .vscode directory suggests that the website is developed on a system that ignores case and so could test work just fine there then break in production.

File naming conventions.  Consistent and unambigous is good.  If the artifact names are out of your control, never mind about this part.  If they're named by a human, please normalize their names.  In particular:
* "jpg", "jpeg", and "JPG" all refer to the same filetype so pick an extension and be consistent.
* Some files have what I guess to be dates in their names.  Please use ISO-8601 formatted dates, which makes them unambiguous, avoids cultural confusion, and makes them sortable.  For example I guess "3-26-19" to mean March 26, 2019; ISO-8601 format would be 2019-03-26.
* Some tools have trouble with filename with spaces in them.  Replace spaces with underscores to avoid problems from naive use of those tools.

Commit:
cleaned up contents of git repo, unused images, artifacts, etc.

Commit:
removed .vscode from repo.

Commit:
renamed Effective Evaluations document so that name does not contain spaces.

Masterpoda commented:
Thanks for looking over the repo! I think I've addressed everything you mentioned. Thankfully much of the name convention violation was by image files that weren't in use yet, so I've removed them for now and will make sure they're named properly if they should be added later. There's no need for them to be on the repo anyway. 

I've added the .vscode file to the gitignore. The other artifact was probably caused by me mistakenly opening one of the documents in openoffice or something. I'll add an entry in gitignore for these kind of artifacts if they become a problem again.

There were also a few document files that had spaces so I fixed those as well.

nickthetait commented:
Congrats on your first merged pull request! I'll deploy the new stuff in the next few days...
